### ISSUE TITLE
Integration test: expected and actual results are swapped in assertions.

### ISSUE THREAD

**Issue by mkulak** [zalando-incubator/zally#520]
[This build](https://travis-ci.org/zalando-incubator/zally/builds/276469851#L3222) failed with:
```
 --- FAIL: TestIntegrationWithLocalYamlFile/integrationWithLocalYamlFile (2.00s)
	Error Trace:	integration_test.go:59
	Error:      	Not equal: 
	            	expected: "Failing because: 7 must violation(s) found"
	            	actual: "Failing because: 6 must violation(s) found"
```

But [in fact](https://github.com/zalando-incubator/zally/blob/ee20b5885a9155774f7ce69d3ab6153a24b58982/cli/zally/integration_test.go#L59) actual string was `"Failing because: 7 must violation(s) found"` and expected  was `"Failing because: 6 must violation(s) found"`.

**TLDR**: in `integration_test.go` arguments to method [assert.Equal](https://godoc.org/github.com/stretchr/testify/assert#Equal) are passed in wrong order.
